src/app/components/ItemDetails.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * TCD Software
+ * Created by Dmitrij Rysanow on 03.03.17.
+ */
+
+import React from "react";
```

Comment:
Please use convention like this `import React, { Component } from 'react';`
You've used this convention in line 7 and that's cool :)